samples/graphene-redis/README.md

```diff
@@ -108,18 +108,14 @@ You can now securely connect to the Redis server using the `redis-cli` and the M
 
 ## Building the Docker image
 
-*Prerequisite*: Graphene is set up and the original Redis example is working.
-
 To marbleize the example we edited [redis-server.manifest.template](redis-server.manifest.template). See comments starting with `MARBLERUN` for explanations of the required changes.
 
-Assuming you have built Graphene in `/graphene` copy the redis-server.manifest.template into the `/graphene/Examples/redis`
-
 ```bash
 cp ./redis-server.manifest.template /graphene/Examples/redis/redis-server.manifest.template
 ```
```

Comment:
```suggestion
```
Should not be needed anymore since the file is copied during `docker build` from the current directory